src/utils/VectorFactory.ts

```diff
@@ -112,6 +115,9 @@ export function wktParseRect(wkt: any) {
  * converts it to a WKT Polygon.
  */
 export function svgPolygonToWKT(svg: any) {
```

Comment:
svg is a string here and in the next two functions.

Reply:
This code was taken from the old website, and I wasn't sure. We'll change it.